filebeat/module/kibana/audit/ingest/pipeline.yml

```diff
@@ -0,0 +1,21 @@
+description: Pipeline for parsing Kibana audit logs
+processors:
+- set:
+    field: event.ingested
+    value: '{{_ingest.timestamp}}'
+- rename:
+    field: '@timestamp'
+    target_field: event.created
+- pipeline:
+    name: '{< IngestPipeline "pipeline-json" >}'
+- set:
```

Comment:
Would it be possible to also add some event.type possibilities? I don't know if we can differentiate between the different alert types for example? https://www.elastic.co/guide/en/ecs/current/ecs-event.html#field-event-type

Reply:
Ah good catch - we record this information in our audit logs, I just failed to move it over in the pipeline